cmd/listen.go

```diff
@@ -118,16 +110,28 @@ func (l *Listen) handlePackets(s *SendPktFeed, wg *sync.WaitGroup) {
 				log.Errorf("%s: Unable to decode: %s", l.iname, errx.Error())
 			}
 
+			// if our interface is non-promisc, learn the client IP
```

Comment:
this whole case statement should be re-examined and see if it can be optimized.  I think we're parsing the packet 3 times ????  Should open a ticket.